src/commands/createFunction.ts

```diff
@@ -30,12 +31,17 @@ const requiredFunctionAppFiles: string[] = [
     path.join('.vscode', 'launch.json') // NOTE: tasks.json is not required if the user prefers to run 'func host start' from the command line
 ];
 
-function validateTemplateName(rootPath: string, name: string | undefined): string | undefined {
+function validateTemplateName(rootPath: string, name: string | undefined, language: string): string | undefined {
```

Comment:
The Java and Node validation have nothing in common and should thus be different functions

Reply:
the beginning ```name``` undefined checking is common I think. Extract the else block to another function?

Reply:
Mmm that's true - no you don't have to move the else to a different function. Okay just rename 'validateFunctionName' to 'validateJavaFunctionName' and that should be good enough

Reply:
Okay